generic/tohil.c

```diff
@@ -834,15 +858,18 @@ tohil_subinterp_is_set(Tcl_Interp *interp)
 //
 // swap to the python subinterpreter associated with this tcl interpreter
 //
-static void
+static PyThreadState *
 tohil_swap_subinterp(Tcl_Interp *interp)
 {
     TohilPyterps *pyterps = (TohilPyterps *)Tcl_GetAssocData(interp, TOHIL_ASSOC_PYTERPS, NULL);
     assert(pyterps != NULL);
-    if (pyterps->child != PyThreadState_Get()) {
+    PyThreadState *prior = PyThreadState_Get();
+    assert(prior != NULL); // i think a python interpreter has to exist to get here
```

Comment:
Yes, in fact Python will abort on us if `PyThreadState_Get()` would return `NULL`. Makes it a hair annoying when trying to print the thread state.

Reply:
Yeah, it will return NULL until the first Python interpreter is created.